src/Views/PointBlueViews.php

```diff
@@ -46,18 +41,67 @@ public function handle()
     {
         $viewName = $this->argument('viewname');
 
-	    if($viewName == 'footer')
-	    {
-		    $this->installFooter();
-	    }
+        $this->installView($viewName);
     }
 
-    private function installFooter()
+    private function installView($viewName)
     {
-	    $footerDestPath = __DIR__ . self::VIEWS_DEST_PATH . self::PARTIALS_DEST_PATH . self::UNIVERSAL_PARTIALS_DEST_PATH;
-	    $footerBladeFilePath = __DIR__ . self::BLADES_PATH . self::FOOTER_BLADE_FILENAME;
-	    mkdir($footerDestPath, 0775);
-	    $footerContents = file_get_contents($footerBladeFilePath);
-	    file_put_contents($footerDestPath . self::FOOTER_BLADE_DEST_FILENAME, $footerContents);
+        $viewDestPath = resource_path('views/partials/universal/');
```

Comment:
`views/partials/universal/` should be a constant. it's used in several place. if we want to change this, it would be helpful to do so in only one spot